src/compiler/LessCompiler.ts

```diff
@@ -29,6 +29,9 @@ export class LessCompiler implements ICompiler {
         }
         if (!this.pathToEntryFile) {
             throw Error(`[${this.pluginName}] cannot determine path to LESS entry file`);
+        } else if (path.basename(mainRepoDir) === 'cplace') {
```

Comment:
Mh we should place that error message somewhere else. I'd probably move it to the overall run behavior and when someone runs the assets compiler *not* `--onlypre` *and* the main folder is *not called* `main` *then* we log a *warning message* that it might not work.
Because during the deployment build the assets compiler is indeed run in a folder that is not called `main` with `--localonly` and it works because the ant build replaces the include path in the less files.

Reply:
ui there's also a TriciaLessCompiler...